core/controllers/suggestion.py

```diff
@@ -139,6 +161,69 @@ def put(self, target_id, suggestion_id):
         self.render_json(self.values)
 
 
+class ReviewableSuggestionsHandler(base.BaseHandler):
+    """Provides all suggestion which can be reviewed by the user for a given
+    suggestion type.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self, target_type, suggestion_type):
+        """Handles GET requests."""
+        try:
+            _check_valid_suggestion_type_and_target_type(
+                target_type, suggestion_type)
+
+            suggestions = suggestion_services.get_reviewable_suggestions(
+                self.user_id, suggestion_type)
+
+            if target_type == suggestion_models.TARGET_TYPE_EXPLORATION:
+                tagret_id_to_details = (
+                    opportunity_services.get_opportunities_by_ids(
+                        [s.target_id for s in suggestions]))
+                self.render_json({
+                    'suggestions': [s.to_dict() for s in suggestions],
+                    'tagret_id_to_details': {
```

Comment:
" target" is misspelled, here and above.

Reply:
Done!